### PR TITLE
Add application matrix

### PR DESCRIPTION
**Pull request by janbrummer**

### CODE REVIEW

File: docs/applications.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+| Application | Upstream |
+| --- | -- |
+| Mozilla Firefox | YES |
+| glib-networking | YES |
+| apt | NO |
+| cURL | NO |
+| wget | NO |
+| libneon | NO |
```
Comment (DimStar77) [inline docs/applications.md:14]: https://build.opensuse.org/package/show/openSUSE:Factory/neon
this is libproxy enabled in openSUSE (no patch)

Comment (DimStar77) [reply docs/applications.md:14]: According to openSUSE's changelog, libproxy support was introduced with version 0.29.3

```
   - added support for system-default proxies:
     ne_session_system_proxy(), implemented using libproxy
```

**Comment by janbrummer** [comment]
> also supporting libproxy already:
> 
>     * libQt5Network
> 
>     * libzypp
> 
>     * seamonkey
> 
>     * signond (KDE)
> 
>     * signon_ui (KDE)
> 
>     * eid-viewer ( https://eid.belgium.be/nl)

Added.

File: docs/applications.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# Applications - Who is using Libproxy?
+
+
+| Application | Upstream |
+| --- | -- |
+| Mozilla Firefox | YES |
+| glib-networking | YES |
+| apt | NO |
+| cURL | NO |
+| wget | NO |
```
[the remark this answers is not in the document]
Comment (janbrummer) [reply docs/applications.md:13]: Added as well.

File: docs/applications.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+# Applications - Who is using Libproxy?
+
+
+| Application | Upstream | Patches |
+| --- | -- | -- |
+| apt | NO | AVAILABLE |
+| cURL | NO | AVAILABLE |
+| eid-viewer | YES | - |
+| glib-networking | YES | - |
+| hexchat | YES | - |
```
Comment (DimStar77) [inline docs/applications.md:13]: indirect only
https://github.com/hexchat/hexchat/commit/25440a07c3b421134b4376d7db3ee4b7ed57ad98

File: docs/applications.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+| libQt5Network | YES | - |
+| libzypp | YES | - |
+| Mozilla Firefox | YES | - |
+| neon | YES | - |
+| openconnect | YES | - |
+| python-requests | NO | AVAILABLE |
+| seamonkey | YES | - |
+| signond | YES | - |
+| signond_ui | YES | - |
+| transmission-remote-gtk | YES | - |
```
Comment (DimStar77) [inline docs/applications.md:23]: at best indirect since https://github.com/transmission-remote-gtk/transmission-remote-gtk/pull/196 (using libsoup3)

**Pull request merged**